### PR TITLE
chore(views): cleaned up php and html for admin pages

### PR DESCRIPTION
**Pull request by jdalsem**

### CODE REVIEW

File: views/default/widgets/online_users/edit.php [hunk 1]
```diff
@@ -9,15 +9,10 @@
 	$num_display = 8;
 }
 
-$params = array(
+$dropdown = elgg_view('input/select', [
```
Comment (hypeJunction) [inline views/default/widgets/online_users/edit.php]: MIght as well change all of these to `elgg_view_input()`

Comment (jdalsem) [inline views/default/widgets/online_users/edit.php]: yeah i need to get used to that new function :)

**Comment by mrclay** [comment]
:-1: for converting readable HTML to `elgg_format_element`.
I'm neutral on the trivial `array()` -> `[]` conversions. They make the blame view harder to use for little gain.

**Comment by hypeJunction** [comment]
IMO, you need to break these down into multiple PRs. I loose focus trying to review.

**Comment by jdalsem** [comment]
Yeah, but then it is hardly worth the effort... as the changes are that trivial

**Comment by hypeJunction** [comment]
I think that's part of the reason to break it down. If it's not worth the effort, maybe better to leave it alone.

**Comment by jdalsem** [comment]
closing this as changes are trivial and there is unwanted usage of elgg_format_element

**Pull request closed without merging**